ui/StatusQ/include/StatusQ/stringutilsinternal.h

```diff
@@ -13,10 +13,12 @@ class StringUtilsInternal : public QObject
 public:
     explicit StringUtilsInternal(QQmlEngine* engine, QObject* parent = nullptr);
 
-    Q_INVOKABLE QString escapeHtml(const QString &unsafe) const;
+    Q_INVOKABLE QString escapeHtml(const QString& unsafe) const;
 
     Q_INVOKABLE QString readTextFile(const QString& filePath) const;
 
+    Q_INVOKABLE QString extractDomainFromLink(const QString& link) const;
```

Comment:
Haven't tested it, but I think `url.host()` will return host name (eg. host.example.com), not the domain (example.com).
```suggestion
    Q_INVOKABLE QString extractHostNameFromLink(const QString& link) const;
```

Reply:
might require renaming it all over the place 👀

Reply:
Ah, merged too soon, but I ended up not using it at all, so all's good in that regard :)